pkgs/tools/misc/jugglinglab/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchgit, jre, makeWrapper, ant, jdk }:
+stdenv.mkDerivation rec {
+  major= "1";
+  minor = "2";
+  version = "${major}.${minor}";
```

Comment:
Looks like the most recent version is 1.2.1 and the program gives me a pop-up saying that there is an update. So I would suggest an update ;)

Minor nit-pick: since you are not really using major/minor version info for anything other than the version, I would collapse this down to `version = "1.2.1"`, as it makes the code easier to follow.